relnotes/config.feature.md

```diff
@@ -0,0 +1,6 @@
+* `swarm.neo.client.mixins.ClientCore`
+
+  A new constructor has been added that accepts a ``Config`` struct instance
```

Comment:
`s/struct/class` in multiple places I guess.

Reply:
Updated.

Reply:
Oh wait, I missed the commit message.

Reply:
Updated.

Reply:
This is still `Config` struct instance.

Reply:
Ha I missed this one completely ;)

Reply:
Updated again.